test/actions/egg-rejected.js

```diff
@@ -31,15 +31,15 @@ describe('rejectEggFromBasket action', () => {
                                             action: "retryCurrentChallenge"
                                           },
                                           rightButton: {
-                                            label: "~BUTTON.NEXT_CASE",
+                                            label: "~BUTTON.NEXT_MISSION",
                                             action: "navigateToNextChallenge"
                                           },
                                           top: undefined,
                                           showAward: true
                                         };
 
     const dispatch = expect.createSpy();
-    const getState = () => ({case: 0, challenge: 0, trial: 0, trials: [{}], authoring: [[{}]] });
+    const getState = () => ({routeSpec: {mission: 0, challenge: 0}, trial: 0, trials: [{}], authoring: [[{}]] });
```

Comment:
Seems like `level` should be included in the `routeSpec`, even though it's apparently not strictly necessary.